cfme/tests/infrastructure/test_config_management.py

```diff
@@ -1,24 +1,23 @@
-
 import fauxfactory
 import pytest
 from cfme.fixtures import pytest_selenium as sel
 from cfme.configure.configuration import Category, Tag
-from cfme.infrastructure.config_management import ConfigManager
-from utils import error
+# from cfme.infrastructure.config_management import ConfigManager, Satellite, AnsibleTower
+from utils import error, version
 from utils.update import update
+from utils.testgen import config_managers, generate
 
 
-@pytest.fixture
-def config_manager_obj(cfme_data):
-    """ Fixture that provides a random config manager (it doesn't set it up)"""
-    cfg_mgr_key = fauxfactory.gen_choice(cfme_data['configuration_managers'].keys())
-    return ConfigManager.load_from_yaml(cfg_mgr_key)
+pytest_generate_tests = generate(config_managers)
 
 
 @pytest.yield_fixture
 def config_manager(config_manager_obj):
     """ Fixture that provides a random config manager and sets it up"""
-    config_manager_obj.create()
+    if config_manager_obj.type == "AnsibleTower":
+        config_manager_obj.create(validate=False)
```

Comment:
Comment on why we dont validate Ansible Tower please. You know why and I know as well but everyone else (who might work with the code) should know as well.